index.bs

```diff
@@ -339,13 +339,60 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 :: A user agent implementing, in conjunction with the underlying platform, the [=Web Authentication API=] and algorithms
     given in this specification, and handling communication between [=authenticators=] and [=[RPS]=].
 
+: <dfn>Credential ID</dfn>
+:: A probabilistically-unique [=byte sequence=] identifying a [=public key credential source=].
+
+    Credential IDs are generated by [=authenticators=] in two forms:
+    1. Bytes with at least 100 bits of entropy, or
+    1. The [=public key credential source=], without its [=public key credential source/id=] item, encrypted so only the owning
+        authenticator can decrypt it.
```

Comment:
In U2F, there is no definition of what is inside a key handle. It is defined as an opaque blob of up to 256 bytes (but private key and AppID are a functional minimum).

I believe webauthn should not try to define what is inside this blob. The contents will depend on the authenticator needs (with or without display, first or second factor, local storage, supported extensions, ...). It's pretty obvious what to include based on what the vendor chooses to support. I believe trying to define it will either impose unnecessary limits or just be stating the obvious.

As for guidance of how it is wrapped: make sure it cannot collide with the requirements of the FIDO security requirements working group.